src/renderer/src/components/editor/OfficePreview.module.css

```
@@ -0,0 +1,175 @@
+   rather than Tailwind utilities. Scoped to a module to avoid leaking globally. */
+
+.officePreview {
+  max-width: 900px;
+  margin: 0 auto;
+  padding: 48px 56px;
+  overflow: auto;
+  color: var(--foreground);
+  background: var(--background);
+  /* Why: Word's default body is Calibri 11pt; Carlito is the metric-compatible open font Linux distros ship. Geist is the Orca sans fallback (--font-sans) per STYLEGUIDE.md §3. */
+  font-family:
+    'Carlito',
```

Stylelint (17.14.0)

[error] Expected no quotes around "Carlito" (font-family-name-quotes)

(font-family-name-quotes)